locale/merge_csl_locales

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import sys
+import urllib.request
+import shutil
+import xml.etree.ElementTree as ET
+import unicodedata
+
+LOCALES_GIT_REF = "bd8d2dbc85713b192d426fb02749475df30f0d2c"
```

Comment:
No need to hard-code this. We'd re-run this whenever we wanted to update it, so it should use the current commit.